synapse/lib/storm.py

```diff
@@ -136,7 +136,87 @@
 
             }
         '''
-    }
+    },
+    {
+        'name': 'view.add',
+        'descr': 'Add a view to the cortex.',
+        'cmdargs': (
+            ('--layer', {'default': [], 'action': 'append', 'help': 'Layers for the view.'}),
```

Comment:
Would this require a `--layer <iden> --layer <iden>` use case?  If so, maybe we should just do --layers and comma split?